distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -19,7 +19,7 @@ The Content Directory Monitor is installed by default with a standard installati
 
 [TIP]
 ====
-If the CDM being configured is WebDav in place, or the directory being monitored is inside ${home_directory}, then adding permissions is not required and this section can be skipped.
+If the CDM being configured is WebDav in place then adding permissions is not required and this section can be skipped.
```

Comment:
❓ Is this for `WebDav` or `In Place` then adding permissions isn't required?

Reply:
WebDav, but WebDav is in place.